pom.xml

```diff
@@ -230,10 +241,17 @@
                 <groupId>com.github.github</groupId>
                 <artifactId>site-maven-plugin</artifactId>
                 <configuration>
-                    <path>${project.version}</path>
+                    <!--<path>${project.version}</path>-->
                     <message>Creating site for ${project.name} ${project.version}</message>
                     <merge>true</merge>
                     <noJekyll>true</noJekyll>
+                    <outputDirectory>${project.build.directory}/mvn-repo</outputDirectory>
+                    <branch>refs/heads/mvn-repo</branch>
+                    <includes>
+                        <include>**/*</include>
+                    </includes>
+                    <repositoryName>dropwizard-elasticsearch</repositoryName>
+                    <repositoryOwner>engagetech</repositoryOwner>
```

Comment:
Please remove changes in the <configuration> block.

Reply:
Sorry @frangz which block?

Reply:
The configuration block.